src/js/components/Marathon.jsx

```diff
@@ -100,14 +101,14 @@ var Marathon = React.createClass({
 
     Mousetrap.bind("shift+,", function () {
       router.navigate("about", {trigger: true});
-    }.bind(this));
+    });
 
     this.startPolling();
   },
 
   componentDidUpdate: function (prevProps, prevState) {
-    if (prevState.activeApp != this.state.activeApp ||
-      prevState.activeTabId != this.state.activeTabId) {
+    if (prevState.activeApp !== this.state.activeApp ||
```

Comment:
I remember that we need the ```!=``` here, because of the comparison off null and undefined, that happen here.